index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" as="style" crossorigin="" href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css">
+    <title>판다마켓</title>
+</head>
+
+<body>
+    <div class="header">
+        <div class="header_contents">
+            <a href="/"><img src="./Group 19.png"></a>
+            <div class="header_login">
+                <a class="link_color" href=".login">로그인
```

Comment:
❗️ 수정요청
여는 태그가 있으면 닫는 태그도 있어야 합니다~

```suggestion
                <a class="link_color" href="./login.html">로그인</a>
```